docs/source/user_guide.md

```diff
@@ -1,3 +1,90 @@
+    </tr>
+    <tr>
+      <td colspan="2" align="center"><a href="./examples#notebook-examples">Notebook</a></td>
+      <td colspan="1" align="center"><a href="./examples">Examples</a></td>
+      <td colspan="1" align="center"><a href="./docs/source/validated_model_list.md">Results</a></td>
+      <td colspan="5" align="center"><a href="https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top.html">Intel oneAPI AI Analytics Toolkit</a></td>
+    </tr>
+  </tbody>
+  <thead>
+    <tr>
+      <th colspan="9">Python-based APIs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td colspan="2" align="center"><a href="./docs/source/quantization.md">Quantization</a></td>
+        <td colspan="3" align="center"><a href="./docs/source/mixed_precision.md">Advanced Mixed Precision</a></td>
+        <td colspan="2" align="center"><a href="./docs/source/pruning.md">Pruning (Sparsity)</a></td> 
+        <td colspan="2" align="center"><a href="./docs/source/distillation.md">Distillation</a></td>
+    </tr>
+    <tr>
+        <td colspan="2" align="center"><a href="./docs/source/orchestration.md">Orchestration</a></td>        
+        <td colspan="2" align="center"><a href="./docs/source/benchmark.md">Benchmarking</a></td>
+        <td colspan="3" align="center"><a href="./docs/source/distributed.md">Distributed Compression</a></td>
+        <td colspan="3" align="center"><a href="./docs/source/export.md">Model Export</a></td>
+    </tr>
+  </tbody>
+  <thead>
+    <tr>
+      <th colspan="9">Neural Coder (Zero-code Optimization)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td colspan="1" align="center"><a href="./neural_coder/docs/PythonLauncher.md">Launcher</a></td>
+        <td colspan="2" align="center"><a href="./neural_coder/extensions/neural_compressor_ext_lab/README.md">JupyterLab Extension</a></td>
+        <td colspan="3" align="center"><a href="./neural_coder/extensions/neural_compressor_ext_vscode/README.md">Visual Studio Code Extension</a></td>
+        <td colspan="3" align="center"><a href="./neural_coder/docs/SupportMatrix.md">Supported Matrix</a></td>
+    </tr>    
+  </tbody>
+  <thead>
+      <tr>
+        <th colspan="9">Advanced Topics</th>
+      </tr>
+  </thead>
+  <tbody>
+      <tr>
+          <td colspan="1" align="center"><a href="./docs/source/adaptor.md">Adaptor</a></td>
+          <td colspan="2" align="center"><a href="./docs/source/tuning_strategies.md">Strategy</a></td>
+          <td colspan="3" align="center"><a href="./docs/source/distillation_quantization.md">Distillation for Quantization</a></td>
+          <td colspan="3" align="center"><a href="./docs/source/metric.md">Metric</a></td>    
+      </tr>
+      <tr>     
+        <td colspan="3" align="center"><a href="./docs/source/objective.md">Objective</a></td>
+        <td colspan="3" align="center"><a href="./docs/source/migration.md">Code Migration</a></td>
```

Comment:
`Code Migration` should be highlight in the very beginning, not in this corner. And it's not an `advanced topic` it should belong to `basic topic`.